src/Vaxis.zig

```diff
@@ -197,9 +209,13 @@ pub fn queryTerminal(self: *Vaxis) !void {
 
     _ = try tty.write(ctlseqs.primary_device_attrs);
     try tty.flush();
+}
 
-    // 1 second timeout
-    std.Thread.Futex.timedWait(&self.query_futex, 0, 1 * std.time.ns_per_s) catch {};
+/// Enable features detected by responses to queryTerminal. This function
+/// is only for use with a custom main loop. Call Vaxis.queryTerminal() if
+/// you are using Loop.run()
+pub fn enableTerminalFeatures(self: *Vaxis) !void {
```

Comment:
Bikeshed: I think we should call this `enableDetectedFeatures`. It's a little more clear that this only applies to *some* features. What do you think?

Reply:
Sure.